packages/squiggle-lang/src/rescript/FunctionRegistry/Library/FR_List.res

```diff
@@ -189,12 +191,14 @@ let library = [
       FnDefinition.make(
         ~name="map",
         ~inputs=[FRTypeArray(FRTypeAny), FRTypeLambda],
-        ~run=(inputs, _, env, reducer) =>
-          switch inputs {
+        ~run=(inputs, _, env, reducer) => {
+          let result = switch inputs {
```

Comment:
Any reason for the change here? Was this just for your debugging?

Reply:
Yes, to better see the access type.